extensions/resteasy-classic/resteasy-mutiny/deployment/src/main/java/io/quarkus/resteasy/mutiny/deployment/ResteasyMutinyProcessor.java

```diff
@@ -8,7 +10,16 @@
 
     @BuildStep
     FeatureBuildItem feature() {
+        warn();
         return new FeatureBuildItem(Feature.RESTEASY_MUTINY);
     }
 
+    void warn() {
+        Logger.getLogger(ResteasyMutinyProcessor.class).warn("The quarkus-resteasy-mutiny extension is deprecated. " +
+                "Switch to RESTEasy Reactive instead.\n" +
+                "This extension adds support for Uni and Multi to RESTEasy 'classic', without using the reactive execution model,"
+                +
+                " as RESTEasy classic does not use it. To properly integrate Mutiny and RESTEasy, use RESTEasy reactive.");
```

Comment:
```suggestion
                " as RESTEasy classic does not use it. To properly integrate Mutiny and RESTEasy, use RESTEasy Reactive. See https://quarkus.io/guides/getting-started-reactive for more detailed instructions");
```

Not sure if the link is the correct one of if we should have a guide explaining how to migrate